plugins/Installation/Controller.php

```diff
@@ -567,15 +570,17 @@ public function getInstallationJs()
 
         CoreVue::addJsFilesTo($files);
 
-        $coreHomeUmd = Development::isEnabled() ? 'CoreHome.umd.js' : 'CoreHome.umd.min.js';
-        $files[] = "plugins/CoreHome/vue/dist/$coreHomeUmd";
+        $files[] = AssetManager\UIAssetFetcher\PluginUmdAssetFetcher::getUmdFileToUseForPlugin('CoreHome');
+        $files[] = AssetManager\UIAssetFetcher\PluginUmdAssetFetcher::getUmdFileToUseForPlugin('Installation');
 
         if (defined('PIWIK_TEST_MODE') && PIWIK_TEST_MODE
             && file_exists(PIWIK_DOCUMENT_ROOT . '/tests/resources/screenshot-override/override.js')) {
             $files[] = 'tests/resources/screenshot-override/override.js';
         }
 
-        return AssetManager::compileCustomJs($files);
+        $result = StaticContainer::get('Piwik\Translation\Translator')->getJavascriptTranslations() . "\n";
```

Comment:
We actually can't include the javascript translations here. The problem is, that it is possible to change the language in the drop down. Doing that will change the language, but the string contained in javascript will still be in the initially loaded language, as the javascript is cached.